portal/migrations/versions/2fdf0e8b8e08_.py

```diff
@@ -0,0 +1,26 @@
+"""empty message
+
+Revision ID: 2fdf0e8b8e08
+Revises: 7b4733a907f9
+Create Date: 2017-08-10 14:32:32.887039
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '2fdf0e8b8e08'
+down_revision = '7b4733a907f9'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_unique_constraint(None, 'scheduled_jobs', ['name'])
```

Comment:
IIRC this will create problems with the upgrade / downgrade loop, when the constraint isn't named.  See some of the other migrations where a unique constraint is added.  Test w/ `flask db downgrade && flask db upgrade`